DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: flipAnalysisOfVariance
 Type: Package
 Title: Functions for computing ANOVA, MANOVA, and related methods
-Version: 1.2.20
+Version: 1.2.21
```

Comment:
This P.R. changes the default behaviour of an exported function? Semantic versioning would bump the version to 1.3.0.